Makefile

```diff
@@ -11,7 +11,7 @@ MAKEFLAGS += -B
 
 DOCKER_RUN := docker run --rm=true --privileged \
     -v $(PWD):/sonic \
-    -it sonic-slave-$(USER)
+    sonic-slave-$(USER)
```

Comment:
Maybe not necessary in this case, I think we can keep '-i'. It is possible that some internal binary will detect environment and run conditional code.